### PR TITLE
add styles for header, fix markup and change url in readme.md

### PR DESCRIPTION
**Pull request by k1NAs**
[DEMO LINK](https://orpheus29.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://k1NAs.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,66 @@
...
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
```
Comment (alexander-ignatow) [inline src/index.html:33]: Please add some line breaks for better readability:

```suggestion
    <header class="header">
    
      <a href="#home" class="header__logo-link">
        <img class="header__logo"
          src="./images/logo.png"
          alt="MOYO icon"
        >
      </a>
      
      <nav class="header__nav nav">
        <ul class="nav__list">
        
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#samsung">
              Samsung
            </a>
          </li>
```
And so on - for all `<li>` elements as well

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+  text-decoration: none;
+}
+
+.header {
+  margin: 0 auto;
+  font: var(--nav-link-fz) Roboto, sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: var(--header-line-height);
```
Comment (alexander-ignatow) [inline src/style.css:27]: I would say it is not the correct place to set line-height
Better move it to more specific selectors like `nav__link`

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,82 @@
-body {
+* {
+  list-style: none;
+  text-decoration: none;
   margin: 0;
+  padding: 0;
+}
```
Comment (alexander-ignatow) [inline src/style.css:6]: Avoid setting values for `*` unless necessary - please move this to more specific selectors

For instance - move margin and padding to `html` or `body`, text decoration to link styles, ec

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,82 @@
...
+  width: 100%;
+  background: var(--hover-link-color);
+  display: block;
+  position: absolute;
+  bottom: 0;
+  border-radius: 5px;
+}
+
+.nav__link:hover {
+  cursor: pointer;
```
Comment (alexander-ignatow) [inline src/style.css:80]: No need to set `cursor:pointer` for the links :)

File: src/index.html [hunk 5]
```diff
@@ -6,9 +6,80 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href="#home" class="header__logo-link">
```
Comment (BudnikOleksii) [inline src/index.html:18]:
```suggestion
    <header class="header">
      <a href="#home" class="header__logo-link">
```
Keep empty lines only between sibling elements, you don't need them between parent and child elements.

**Review by BudnikOleksii — APPROVED** [review]
Good job🚀
Just remember to keep empty lines only between sibling elements, you don't need them between parent and child elements.

**Pull request closed without merging**